client-course-schedulizer/src/utilities/hooks/addSectionHooks.ts

```diff
@@ -118,7 +118,7 @@ const scrollToUpdatedSection = (newCourse: Course, newSection: Section) => {
     });
   });
   if (newElement && newElement.parentElement?.parentElement) {
-    newElement.scrollIntoView({ behavior: "smooth", block: "nearest", inline: "nearest" });
+    newElement.scrollIntoView(false);
```

Comment:
Is this change on purpose?

Reply:
Yes, I forgot that I did this, but this is to make it compatible with Safari, see:

![image](https://user-images.githubusercontent.com/51130302/114022886-3b96ce80-9840-11eb-9a93-776bbb4b2c70.png)

Safari doesn't support the `scrollIntoViewOptions` parameter, but they do support the `alignToTop` boolean parameter. Unfortunately, this will cause the scroll into view to not be smooth, but I think it's worth it. (Also the scroll into view for the faculty schedule still has the `scrollIntoViewOptions` parameter, but I figure if that one doesn't work on Safari it isn't the biggest deal)

Reply:
Gotcha, thanks!